src/Features/Core/Portable/UnusedReferences/Reference.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.CodeAnalysis.UnusedReferences
+{
+    internal class Reference
+    {
+        /// <summary>
+        /// Indicates the type of reference.
+        /// </summary>
+        public ReferenceType Type { get; }
+
+        /// <summary>
+        /// Uniquely identifies the reference.
+        /// </summary>
+        /// <remarks>
+        /// Should match the Include or Name attribute used in the project file.
+        /// </remarks>
+        public string ItemSpec { get; }
```

Comment:
Specification vs spec?

Reply:
I believe the MSBuild term for this concept is ItemSpec, but I have no problem with ItemSpecification.